pwndbg/commands/context.py

```diff
@@ -155,8 +157,16 @@ def context_source():
         if not source or source.startswith('1\t'):
             return []
 
+        # highlight the current code line
+        source_lines = source.splitlines()
+        if pwndbg.config.highlight_source:
+            for i in range(len(source_lines)):
```

Comment:
Nevermind, the code above already does this.

It appears there's not a way to get the actual line *contents* out via standard API calls :(